backend/src/useCases/compliance/handleSubmitForm.ts

```diff
@@ -0,0 +1,100 @@
+import { Request, Response } from 'express';
+import mongoose from 'mongoose';
+import { ComplianceDbRepository } from "myTypes";
+import { appConfig } from '../../config';
+import axios from 'axios';
+
+export default class SubmitFormRequestHandler {
+    private repository: ComplianceDbRepository;
+
+    constructor(
+        private req: Request,
+        private res: Response,
+        repository: ComplianceDbRepository
+    ) {
+        this.repository = repository;
+    }
+
+    async submitForm(): Promise<Response> {
```

Comment:
As we are changing the draft to submitted status we should ensure that jira ticket has been created. therefore whole submitForm() should be transacitonal